src/main/java/software/amazon/encryption/s3/S3AsyncEncryptionClient.java

```diff
@@ -257,6 +263,7 @@ public static class Builder {
         private boolean _enableMultipartPutObject = false;
         private Provider _cryptoProvider = null;
         private SecureRandom _secureRandom = new SecureRandom();
+        private long _bufferSize = 0;
```

Comment:
nit: I prefer `-1` as the value for "not used" (see how `contentLength` is handled elsewhere)

Reply:
ack!